docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.8'
+
+services:
+  db:
+    image: mysql:8
+    volumes:
+      - ./mysql:/var/lib/mysql
+
+  api:
+    depends_on:
+      - db
+    restart: always
+    env_file:
+      - .env
+    ports:
+      - ${NODE_PORT}:${NODE_PORT}
```

Comment:
Here we should always expose the same port internally, ie:

In the Dockerfile
```
EXPOSE 3200
```

docker-compose
```
${NODE_PORT}:3200
```